.gitignore

```diff
@@ -31,3 +31,6 @@ tree/master/cloud/src/solution/*/pkgobj/
 /tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Web/wwwroot/js
 /tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Service/Microsoft.Legal.MatterCenter.ServiceTest/bin/Debug/net451
 /tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Service/Microsoft.Legal.MatterCenter.ServiceTest/obj/Debug/net451
+/tree/master/cloud/src/solution/ManagedColumn.cs

```

Comment:
This file shouldn't be ignored. If this get ignored, the solution wont compile. Please remove this file from gitignore